src/Build.UnitTests/Graph/IsolateProjects_Tests.cs

```diff
@@ -474,7 +495,7 @@ public void SkippedTargetsShouldNotTriggerCacheMissEnforcement()
 </Project>
 ".Cleanup()).Path;
 
-            _buildParametersPrototype.IsolateProjects.ShouldBeTrue();
+            Assert.Equal(IsolateProjects.True, _buildParametersPrototype.IsolateProjects);
```

Comment:
Nit: This test uses Shouldly, so this like probably "should" (pun intended) be `_buildParametersPrototype.IsolateProjects.ShouldBe(IsolateProjects.True);`